playbooks/roles/prereqs/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+# tasks file for prereqs role
+
+# Check and enable nested virtualization is necessary
+- import_tasks: setup_nested_virt.yml
+
+# install the kvm plugin
+- import_tasks: install_kvm_plugin.yml
+
+# Install kvm plugin and management libvirt
+- import_tasks: install_virtual_reqs.yml
```

Comment:
`install_virtuall_req` nned to be imported as a 2nd step otherwise we try to strart libvirtd even thought it was not installed.

```
TASK [prereqs : Start and enable libvirtd] *************************************
fatal: [sgi.example.com]: FAILED! => {"changed": false, "msg": "Could not find the requested service libvirtd: host"}
        to retry, use: --limit @/home/user/projects/contra-env-setup/playbooks/setup.retry

PLAY RECAP *********************************************************************sgi.example.com : ok=18   changed=11   unreachable=0    failed=1
```

```
diff --git a/playbooks/roles/prereqs/tasks/main.yml b/playbooks/roles/prereqs/tasks/main.yml
index c64a253..bd2c47c 100644
--- a/playbooks/roles/prereqs/tasks/main.yml
+++ b/playbooks/roles/prereqs/tasks/main.yml
@@ -4,8 +4,8 @@
 # Check and enable nested virtualization is necessary
 - import_tasks: setup_nested_virt.yml
 
+# Install kvm plugin and management libvirt
+- import_tasks: install_virtual_reqs.yml
+
 # install the kvm plugin
 - import_tasks: install_kvm_plugin.yml
-
-# Install kvm plugin and management libvirt
-- import_tasks: install_virtual_reqs.yml
```